### PR TITLE
app/utils/netcmd_netmon: Add wifi command to extract wifi info

### PR DESCRIPTION
**Pull request by SeonghoByeon**
- Add wifi command to extract status and conneciton info

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [ff017fbe9a4539dcc32a41b35329adb768675cb2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [ff017fbe9a4539dcc32a41b35329adb768675cb2] - Code Rule Check OK.

File: apps/system/utils/netcmd_netmon.c [hunk 1]
```diff
@@ -201,8 +205,53 @@ int cmd_netmon(int argc, char **argv)
...
+		printf("=======================================================================\n");
+		printf("CONN    CONNFAIL    DISCONN    RECONN    SCAN    SOFTAP    JOIN    LEFT\n");
+		printf("%-8d%-12d%-11d%-10d", stats.connect, stats.connectfail, stats.disconnect, stats.reconnect);
+		printf("%-8d%-10d%-8d%-8d\n", stats.scan, stats.softap, stats.joined, stats.left);
+		printf("=======================================================================\n");
+
+		printf("Connection INFO.\n");
+		res = wifi_manager_get_info(&info);
+		if (res != WIFI_MANAGER_SUCCESS) {
+			printf("Get Wi-Fi Manager state info failed\n");
```
Comment (davidfather) [inline apps/system/utils/netcmd_netmon.c]: not state info but connection info.

Comment (SeonghoByeon) [reply apps/system/utils/netcmd_netmon.c]: Thanks for the comment. Modification completed.

**Comment by seinfra** [comment]
Target : [09e00dcba20701dcbf2a5dbe64ef435da7ce69dc] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [09e00dcba20701dcbf2a5dbe64ef435da7ce69dc] - Code Rule Check OK.

**Pull request merged**